homeassistant/components/vicare/sensor.py

```diff
@@ -132,6 +133,134 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="hotwater_gas_summary_consumption_heating_lastsevendays",
+        name="Hot water gas consumption last seven days",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterLastSevenDays(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="power_summary_consumption_heating_currentday",
+        name="Power consumption of gas heating current day",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        value_getter=lambda api: api.getPowerSummaryConsumptionHeatingCurrentDay(),
+        unit_getter=lambda api: api.getPowerSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="power_summary_consumption_heating_currentmonth",
+        name="Power consumption of gas heating current month",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        value_getter=lambda api: api.getPowerSummaryConsumptionHeatingCurrentMonth(),
+        unit_getter=lambda api: api.getPowerSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="power_summary_consumption_heating_currentyear",
+        name="Power consumption of gas heating current year",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        value_getter=lambda api: api.getPowerSummaryConsumptionHeatingCurrentYear(),
+        unit_getter=lambda api: api.getPowerSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="power_summary_consumption_heating_lastsevendays",
+        name="Power consumption of gas heating last seven days",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        value_getter=lambda api: api.getPowerSummaryConsumptionHeatingLastSevenDays(),
+        unit_getter=lambda api: api.getPowerSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="power_dhw_summary_consumption_heating_currentday",
+        name="Power consumption of hot water gas heating current day",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        value_getter=lambda api: api.getPowerSummaryConsumptionDomesticHotWaterCurrentDay(),
+        unit_getter=lambda api: api.getPowerSummaryConsumptionDomesticHotWaterUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="power_dhw_summary_consumption_heating_currentmonth",
+        name="Power consumption of hot water gas heating current month",
```

Comment:
All these sensors are using "Power" in their name/description, but are actually Energy sensors (and thus not Power).